sdk/storage/storage-queue/src/generated/src/storageClientContext.ts

```diff
@@ -11,7 +11,7 @@
 import * as coreHttp from "@azure/core-http";
 
 const packageName = "azure-storage-queue";
-const packageVersion = "1.0.0";
+const packageVersion = "12.0.0-preview.5";
```

Comment:
What's silly is that this constant doesn't actually get used. Yes it is set on the options bag below, but that option is only used when the default pipeline policies are used in ServiceClient. Since Storage passes its own policy factories, it's not used.

Reply:
Would it make since for one of these to just default to the other to avoid the duplicated versions?

Reply:
This is generated code.
Autorest fails to get the version from package.json and hence it is defaulted to 1.0.0
![image](https://user-images.githubusercontent.com/10452642/67707947-9c965380-f978-11e9-8d12-171731fbeccd.png)
This will be overridden if code is generated again.

@xirzec, please log an issue.

cc - @ramya-rao-a